### PR TITLE
Fix: decode resource page content

### PR DESCRIPTION
**Pull request by alexanderleegs**
This PR fixes a bug in which resource page content was not being decoded, while collection page and unlinked page content were, resulting in resource page content not being displayed.

### CODE REVIEW

**Review by gweiying — APPROVED** [review]
lgtm

File: routes/resourcePages.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const express = require('express');
 const router = express.Router();
+const base64 = require('base-64');
```
Comment (gweiying) [inline routes/resourcePages.js:3]: typo Base64

File: routes/resourcePages.js [hunk 2]
```diff
@@ -68,7 +69,9 @@ async function readResourcePage (req, res, next) {
   const IsomerFile = new File(accessToken, siteName)
   const resourcePageType = new ResourcePageType(resourceRoomName, resourceName)
   IsomerFile.setFileType(resourcePageType)
-  const { sha, content } = await IsomerFile.read(pageName)
+  const { sha, content: encodedContent } = await IsomerFile.read(pageName)
+
+  const content = Base64.decode(encodedContent)
```
Comment (gweiying) [inline routes/resourcePages.js:74]: should this be `base64` not `Base64`

**Review by gweiying — CHANGES_REQUESTED** [review]
small typo

File: routes/resourcePages.js (hunk 2 above)
Comment (alexanderleegs) [reply routes/resourcePages.js]: Resolved in https://github.com/isomerpages/isomercms-backend/pull/138/commits/553ea4055480ba67a59486bde57d4e9b508a82c1

**Review by gweiying — APPROVED** [review]
lgtm

**Review by kwajiehao — CHANGES_REQUESTED** [review]
This LGTM, however saving resource pages on `EditPage` does not work

**Comment by alexanderleegs** [comment]
This PR is superceded by #139, which covers the refactoring of base64 encoding/decoding, which should also fix the saving of resource pages on `EditPage`.

**Pull request closed without merging**